frontend/app/src/components/Sidebar/Sidebar.tsx

```diff
@@ -284,7 +284,6 @@ const Sidebar: React.FC<SidebarProps> = ({
   // If header decoration visible, move sidebar down so decoration doesn't go below it
   const sidebarAdjust = !isEmbedded && headerDecorationVisible()
 
-  // The tabindex is required to support scrolling by arrow keys.
```

Comment:
Removing this note as it is legacy from st version `0.47.0` (see [here](https://github.com/streamlit/streamlit/blob/0.47.0/frontend/src/components/core/Sidebar/Sidebar.tsx)). The `tabIndex` was removed in `0.48.0`